spacewalk/package/spacewalk.spec

```diff
@@ -24,7 +24,7 @@
 %global postgresql /usr/bin/psql
 %endif
 %if !0%{?is_opensuse}
-%define with_oracle     1
+%define with_oracle     0
```

Comment:
I am not sure if this would work.
The rpm "with" mechanism just check if the variable with_oracle is defined. Even with value 0 it is defined.

I would remove the whole `%if` block